CRM/Member/Form/MembershipRenewal.php

```diff
@@ -594,7 +591,7 @@ protected function submit() {
     if ($contributionRecurID) {
       $membershipParams['contribution_recur_id'] = $contributionRecurID;
     }
-    $membership = $this->processMembership($membershipParams, $renewalDate, $numRenewTerms, $pending);
+    $membership = $this->processMembership($membershipParams, $this->getSubmittedValue('renewal_date'), $numRenewTerms, $pending);
```

Comment:
Oh yes, I forgot we can use getSubmittedValue() :-)

Reply:
@mattwire I try to use it where possible because it clarifies whether we are using what was submitted or whether we need to trawl back to see if it was altered along the way